community/dnscrypt-proxy/dnscrypt-proxy.initd

```diff
@@ -4,17 +4,18 @@
 
 command="/usr/bin/dnscrypt-proxy"
 command_args="${DNSCRYPT_OPTS:--config /etc/dnscrypt-proxy/dnscrypt-proxy.toml}"
-command_user="${DNSCRYPT_USER:-dnscrypt}:${DNSCRYPT_GROUP:-dnscrypt}"
+dir_owner="${DNSCRYPT_USER:-dnscrypt}:${DNSCRYPT_GROUP:-dnscrypt}"
```

Comment:
What is the reason why you changed command_user to dir_owner? 
According with OpenRC manual, this is an option of OpenRC, not a simple variable: 
https://github.com/OpenRC/openrc/blob/master/man/openrc-run.8

Reply:
True -- this is to accommodate the supervisor.  In this new way of running, the supervisor program runs as root, and starts a child process as the user specified in the config file.   I used dir_owner as I still want to run the directory permission checks later in the file.   We can't run the supervisor as an unprivileged user.    Happy to try for a different method if you have suggestions...

Reply:
Ok, yes you mentioned that before. I don't know what is the "standard" way to run processes in this way, actually. Perhaps @ncopa can comment on this.